plugins/gateway/kong/README.md

```diff
@@ -1,5 +1,75 @@
 ## APIClarity Kong plugin
 
+### Prerequisite
+* APIClarity backend is running
+* Kong gateway is running in your K8s cluster, and configured correctly.
+
+If you just want to experience with it, and you don't have kong, you can take these few quick steps that will help you to quickly setup a running environment:
+1. Deploy sock-shop app:
+    ```shell
+       kubectl create namespace sock-shop
+       
+       kubectl apply -f https://raw.githubusercontent.com/microservices-demo/microservices-demo/master/deploy/kubernetes/complete-demo.yaml
+    ```
+2. Deploy Kong:
+    - Using helm3:
```

Comment:
from https://docs.konghq.com/kubernetes-ingress-controller/2.2.x/deployment/k4k8s/#helm the following is also needed.
```
$ helm repo add kong https://charts.konghq.com
$ helm repo update
```